geomesa-blobstore/geomesa-blobstore-api/src/main/scala/org/locationtech/geomesa/blob/api/BlobstoreServlet.scala

```diff
@@ -0,0 +1,48 @@
+package org.locationtech.geomesa.blob.api
+
+import org.locationtech.geomesa.accumulo.data.{AccumuloDataStore, AccumuloDataStoreFactory}
+import org.locationtech.geomesa.blob.core.AccumuloBlobStore
+import org.locationtech.geomesa.web.core.GeoMesaScalatraServlet
+import org.scalatra.{NotFound, Ok}
+
+import scala.collection.JavaConversions._
+
+class BlobstoreServlet extends GeoMesaScalatraServlet {
+  override def root: String = "blob"
+
+  var abs: AccumuloBlobStore = null
+
+  post("/ds/:alias") {
+
+    println("In ds registration method")
+
+    val dsParams = datastoreParams
+    val ds = new AccumuloDataStoreFactory().createDataStore(dsParams).asInstanceOf[AccumuloDataStore]
+
+    if (ds == null) {
```

Comment:
Might be nice to use geoserver persistence so that the ds will persist between restarts. See:
https://github.com/locationtech/geomesa/blob/master/geomesa-web/geomesa-web-data/src/main/scala/org/locationtech/geomesa/web/analytics/AnalyticEndpoint.scala#L32
and
https://github.com/locationtech/geomesa/blob/master/geomesa-web/geomesa-web-data/src/main/resources/applicationContext.xml#L16-L35